modules/distribution-zones/src/testFixtures/java/org/apache/ignite/internal/distributionzones/DistributionZonesTestUtil.java

```diff
@@ -113,11 +113,40 @@ public static CompletableFuture<Integer> createZone(
      * @return A future, which will be completed, when create operation finished.
      */
     public static CompletableFuture<Integer> createZone(
-            DistributionZoneManager zoneManager, String zoneName,
-            int partitions, int replicas) {
+            DistributionZoneManager zoneManager,
+            String zoneName,
+            int partitions,
+            int replicas
+    ) {
         return createZone(zoneManager, zoneName, partitions, replicas, null);
     }
 
+    /**
+     * Creates a distribution zone in the configuration.
+     *
+     * @param distributionZoneManager Distributed zone manager.
+     * @param zoneName Zone name.
+     * @param dataNodesAutoAdjustScaleUp Timeout in seconds between node added topology event itself and data nodes switch,
+     *         {@code null} if not set.
+     * @param dataNodesAutoAdjustScaleDown Timeout in seconds between node left topology event itself and data nodes switch,
+     *         {@code null} if not set.
+     * @param filter Nodes filter, {@code null} if not set.
+     */
+    public static void createZone(
```

Comment:
It's confusing to have both async and sync createZone methods with same createZone name. Do we really need async ones in tests?

Reply:
Async method will leave after switch to the catalog.
I propose to leave it as it is, I can change it to synchronous, wdyt?

Reply:
Not sure that I get it. Do we use the async nature of given method somewhere? If not please change it to synchronous. The main point here is to have consistent contract.

Reply:
fix it